template/.github/workflows/wheels.yml

```diff
@@ -13,24 +13,28 @@ jobs:
     if: startsWith(github.ref, 'refs/tags/v')
 
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v5
         with:
           fetch-depth: 0
+          fetch-tags: true
 
-      - uses: prefix-dev/setup-pixi@v0.8.10
+      - uses: prefix-dev/setup-pixi@v0.9.3
         with:
-          pixi-version: v0.48.2
+          pixi-version: v0.59.0
           frozen: true
           cache: true
-          cache-write: {% raw %}${{ github.ref == 'refs/heads/main' }}{% endraw %}
+          cache-write: false
```

Comment:
Agree. This is such an important step and not done often, that better don't use the cache.